setup/ecr.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "aws_ecr_repository" "repo" {
+  name = local.name_prefix
+
+  encryption_configuration {
+    encryption_type = "AES256"
+  }
+
+  image_scanning_configuration {
+    scan_on_push = var.is_production
```

Comment:
null isn't a valid state for is_production in this case right?

Reply:
The `is_production` var is required, so the only valid values are true or false.  If not set, it will default to false.